.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "err.h": "c"
+    }
+}
```

Comment:
@openssl should really extend .gitignore  like this :
````/.vscode/**````

Reply:
Please don't include a .vscode settings file